packages/nextjs/app/builders/0x2723A2756ecb99b3B50f239782876fB595728AC0/page.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
```

Comment:
We could remove "use client" too, checkout https://github.com/BuidlGuidl/batch4.buidlguidl.com/pull/13#discussion_r1557574281 🙌

Reply:
Done